backend/src/test/java/trn/logistics/knapsack/PdfExporterTest.java

```diff
@@ -0,0 +1,50 @@
+package trn.logistics.knapsack;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import trn.logistics.knapsack.dto.KnapsackSolution;
+import trn.logistics.knapsack.dto.Material;
+import trn.logistics.knapsack.dto.MaterialType;
+import trn.logistics.knapsack.dto.Vehicle;
+import trn.logistics.knapsack.exporter.PdfExporter;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
+public class PdfExporterTest {
+    @Test
+    void createPdfFromKnapsackTest() throws IOException {
+        KnapsackSolution kns = new KnapsackSolution();
+        Vehicle v1 = new Vehicle(400, 3500);
```

Comment:
Just curious, but why not just name it `vehicle`?
Same applies to the PdfExporter below.

With names like this a developer is forced to read the complete code to understand what class is used later.
With names like `vehicle` or `pdfExporter` the context is more clear and the code is easier to understand